app.py

```diff
@@ -0,0 +1,22 @@
+import os
+import pydevd_pycharm
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    msg = 'Hello World!'
+    return msg
+
+def attach():
+  if os.environ.get('WERKZEUG_RUN_MAIN'):
+    print('Connecting to debugger...')
+    pydevd_pycharm.settrace('0.0.0.0', port=3500, stdoutToServer=True, stderrToServer=True)
+
+if __name__ == '__main__':
+  print('Starting hello-world server...')
+  # comment out to use Pycharm's remote debugger
```

Comment:
should we use an envvar instead?

Reply:
I prefer to do it this way since it has nothing to do with Okteto and it's more inline with the pycharm official demos.